SPECS/kernel/config

```diff
@@ -1548,7 +1547,6 @@ CONFIG_NET_SCH_TBF=m
 CONFIG_NET_SCH_ETF=m
 # CONFIG_NET_SCH_TAPRIO is not set
 CONFIG_NET_SCH_GRED=m
-CONFIG_NET_SCH_DSMARK=m
```

Comment:
Removed in commit [6e5e93fd51fea74844cb741af15fdc359ea8712e](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/6e5e93fd51fea74844cb741af15fdc359ea8712e)